### PR TITLE
This is handling of loganalyzer embedded in admin dashboard.

### PR DESCRIPTION
**Pull request by ksdperera**
Adding tenant aware log filtering mechanism in loganalyzer proxy.
Embedded gadgets inside the admin dashboard.

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin-dashboard/site/blocks/loganalyzer-dashboard/ajax/loganalyzer-dashboard.jag [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<%
+include("/jagg/jagg.jag");
```
Comment (keizer619) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin-dashboard/site/blocks/loganalyzer-dashboard/ajax/loganalyzer-dashboard.jag]: @ksdperera  Add License header

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin-dashboard/site/blocks/loganalyzer/ajax/loganalyzer.jag [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<%
+include("/jagg/jagg.jag");
```
Comment (keizer619) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin-dashboard/site/blocks/loganalyzer/ajax/loganalyzer.jag]: @ksdperera  Add License header

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin-dashboard/site/blocks/loganalyzer/block.jag [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<%
+jagg.block("loganalyzer", {
```
Comment (keizer619) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin-dashboard/site/blocks/loganalyzer/block.jag]: @ksdperera  Add License header

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin-dashboard/site/pages/loganalyzer.jag [hunk 4]
```diff
@@ -0,0 +1,43 @@
+<%
+include("/jagg/jagg.jag");
```
Comment (keizer619) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin-dashboard/site/pages/loganalyzer.jag]: @ksdperera  Add License header

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin-dashboard/site/themes/wso2/templates/loganalyzer/css/custom.css [hunk 5]
```diff
@@ -0,0 +1,552 @@
+/*
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (keizer619) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin-dashboard/site/themes/wso2/templates/loganalyzer/css/custom.css]: @ksdperera update License header year

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin-dashboard/site/themes/wso2/templates/loganalyzer/css/custom.css [hunk 6]
```diff
@@ -0,0 +1,552 @@
...
+    overflow-y: auto;
+}
+
+.page-content-wrapper.fixed .sidebar-wrapper.sidebar-nav-sub {
+    height: 100%;
+    z-index: 1000000;
+    background: #272c30;
+}
+
+/*.sidebar-nav-sub:before {*/
```
Comment (keizer619) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin-dashboard/site/themes/wso2/templates/loganalyzer/css/custom.css]: @ksdperera  Remove these commented line if they are not required

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin-dashboard/site/themes/wso2/templates/loganalyzer/css/custom.css [hunk 7]
```diff
@@ -0,0 +1,552 @@
...
+
+[data-side=left] .close-handle.close-sidebar {
+    right: -30px;
+}
+
+[data-side=right] .close-handle.close-sidebar {
+    left: -30px;
+}
+
+/*.sidebar-nav > .sidebar-brand, .sidebar-brand {*/
```
Comment (keizer619) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin-dashboard/site/themes/wso2/templates/loganalyzer/css/custom.css]: @ksdperera  Remove these commented lines if they are not required

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin-dashboard/site/themes/wso2/templates/loganalyzer/initializer.jag [hunk 8]
```diff
@@ -0,0 +1,16 @@
+<%
+jagg.initializer("loganalyzer", {
```
Comment (keizer619) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin-dashboard/site/themes/wso2/templates/loganalyzer/initializer.jag]: @ksdperera  Add License header

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin-dashboard/site/themes/wso2/templates/loganalyzer/template.jag [hunk 9]
```diff
@@ -0,0 +1,271 @@
+<% jagg.template("loganalyzer", function(inputs, outputs, jagg) {
```
Comment (keizer619) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin-dashboard/site/themes/wso2/templates/loganalyzer/template.jag]: @ksdperera  Add License header

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin-dashboard/site/blocks/loganalyzer-dashboard/ajax/loganalyzer-dashboard.jag (hunk 1 above)
Comment (ksdperera) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin-dashboard/site/blocks/loganalyzer-dashboard/ajax/loganalyzer-dashboard.jag]: Hi Tharik,

This file is no longer use and we have removed this by the commits in : https://github.com/wso2/carbon-apimgt/pull/2429

Thanks
SajithD

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin-dashboard/site/themes/wso2/templates/loganalyzer/css/custom.css (hunk 5 above)
Comment (ksdperera) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin-dashboard/site/themes/wso2/templates/loganalyzer/css/custom.css]: Hi Tharik,

This issue is fixed by the commits in : https://github.com/wso2/carbon-apimgt/pull/2906

Thanks
SajithD

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin-dashboard/site/themes/wso2/templates/loganalyzer/css/custom.css (hunk 6 above)
Comment (ksdperera) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin-dashboard/site/themes/wso2/templates/loganalyzer/css/custom.css]: Hi Tharik,

This issue is fixed by the commits in : #2906

Thanks
SajithD

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin-dashboard/site/themes/wso2/templates/loganalyzer/css/custom.css (hunk 7 above)
Comment (ksdperera) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin-dashboard/site/themes/wso2/templates/loganalyzer/css/custom.css]: Hi Tharik,

This issue is fixed by the commits in : #2906

Thanks
SajithD

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin-dashboard/site/blocks/loganalyzer/ajax/loganalyzer.jag (hunk 2 above)
Comment (ksdperera) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin-dashboard/site/blocks/loganalyzer/ajax/loganalyzer.jag]: Hi Tharik,

This license header fixes by the commits in https://github.com/wso2/carbon-apimgt/pull/2906

Thanks,
SajithD

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin-dashboard/site/blocks/loganalyzer/block.jag (hunk 3 above)
Comment (ksdperera) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin-dashboard/site/blocks/loganalyzer/block.jag]: Hi Tharik,

This license header fixes by the commits in https://github.com/wso2/carbon-apimgt/pull/2906

Thanks,
SajithD

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin-dashboard/site/pages/loganalyzer.jag (hunk 4 above)
Comment (ksdperera) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin-dashboard/site/pages/loganalyzer.jag]: Hi Tharik,

This license header fixes by the commits in https://github.com/wso2/carbon-apimgt/pull/2906

Thanks,
SajithD

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin-dashboard/site/themes/wso2/templates/loganalyzer/initializer.jag (hunk 8 above)
Comment (ksdperera) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin-dashboard/site/themes/wso2/templates/loganalyzer/initializer.jag]: Hi Tharik,

This license header fixes by the commits in https://github.com/wso2/carbon-apimgt/pull/2906

Thanks,
SajithD

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin-dashboard/site/themes/wso2/templates/loganalyzer/template.jag (hunk 9 above)
Comment (ksdperera) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin-dashboard/site/themes/wso2/templates/loganalyzer/template.jag]: Hi Tharik,

This license header fixes by the commits in https://github.com/wso2/carbon-apimgt/pull/2906

Thanks,
SajithD

**Pull request merged**